### PR TITLE
Fixup for commit "f090029 Add new test cases for migration."

### PR DESCRIPTION
**Pull request by fangge1212**
Signed-off-by: Fangge Jin <fjin@redhat.com>

### CODE REVIEW

**Comment by fangge1212** [comment]
@dzhengfy

I modified some errors in commit "f090029 Add new test cases for migration", please help me review

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate_vm_cfg.py [hunk 1]
```diff
@@ -353,14 +362,14 @@ def cleanup_vm(vm, vm_name='', uri=''):
...
                 result_check_pass = False
                 logging.error("Src vm should be %s after offline migration" % src_vm_status)
                 logging.debug("Src vm state is %s" % vm.state())
 
         if live_migration:
             if not undefinesource and src_vm_cfg == "persistent":
-                if not obj_migration.check_vm_state(vm.name, "shut off", vm.connect_uri):
+                if not obj_migration.check_vm_state(vm.name, "shut off", uri=vm.connect_uri):
                     result_check_pass = False
                     logging.error("Src vm should be shutoff after live migration")
```
Comment (Yingshun) [inline libvirt/tests/src/virsh_cmd/domain/virsh_migrate_vm_cfg.py]: typo in line 377, resulst - > result

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate_vm_cfg.py [hunk 2]
```diff
@@ -140,17 +146,21 @@ def cleanup_vm(vm, vm_name='', uri=''):
             # Add tls_obj to objs_list
             objs_list.append(tls_obj)
 
+        # Enable libvirtd remote connection transport port
+        if transport_port:
+            obj_migration.migrate_pre_setup(dest_uri, params, ports=transport_port)
```
Comment (Yingshun) [inline libvirt/tests/src/virsh_cmd/domain/virsh_migrate_vm_cfg.py]: We'd better define a variable for 'ports' to reduce the code lines.It still used in finally block.

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate_vm_cfg.py (hunk 1 above)
Comment (fangge1212) [reply libvirt/tests/src/virsh_cmd/domain/virsh_migrate_vm_cfg.py]: updated

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate_vm_cfg.py (hunk 2 above)
Comment (fangge1212) [reply libvirt/tests/src/virsh_cmd/domain/virsh_migrate_vm_cfg.py]: Updated. Thanks for your advice.

**Pull request merged**